packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -843,8 +875,10 @@ class _ViewContentState extends State<_ViewContent> {
       return MediaQuery.removePadding(
         context: context,
         removeTop: true,
-        child: ListView(
-          children: suggestions.toList()
+        child: ListView.builder(
+          shrinkWrap: true,
```

Comment:
Should this always be true?

Reply:
I thought it was weird to keep it always true but it renders correctly in all states. The Flexible widget outside seems to take precedence.

Reply:
Also passing `shrinkWrap` to the `viewBuilder` would probably be a breaking change

Reply:
Do we still need shrinkwrap here? If it works without, it is less expensive to not use it.

Reply:
I double checked and the `shrinkWrap` property is necessary

Reply:
Ok, thanks for checking. I apologize I have forgotten since we originally discussed. Why is it necessary? There are performance concerns when using shrinkwrap.

Reply:
Without it the ListView takes up all available space, which this PR is aiming to fix